### PR TITLE
DOC: update 1.3.2 release notes

### PR DESCRIPTION
**Pull request by tylerjereddy**
* update the release notes for SciPy `1.3.2`
using `authors.py`, `gh_lists.py`, and a little
manual curation, after adjusting milestones
on the appropriate GitHub issues/PRs

### CODE REVIEW

File: doc/release/1.3.2-notes.rst [hunk 1]
```diff
@@ -10,8 +10,58 @@ support for Python 3.8.
 Authors
 =======
 
+* CJ Carey
+* Martin Gauch +
+* Ralf Gommers
+* Matt Haberland
+* Huize +
```
Comment (tylerjereddy) [inline doc/release/1.3.2-notes.rst]: @huizew You may like for us to update the mailmap file if you'd like your name displayed differently in our release notes?

Comment (huizew) [reply doc/release/1.3.2-notes.rst]: @tylerjereddy Thanks. I would like to use full name `Huize Wang`.

**Review by rgommers — APPROVED** [review]

File: doc/release/1.3.2-notes.rst (hunk 1 above)
Comment (tylerjereddy) [reply doc/release/1.3.2-notes.rst]: Ok,  updated the mailmap & regenerated the relnotes accordingly.

**Comment by rgommers** [comment]
Merged, thanks Tyler

**Pull request merged**